example/api.rsh

```diff
@@ -1,2 +1,6 @@
 /// Prints a friendly greeting to stdout.
 fn hello_world();
+
+/// Returns a future that prints a friendly
+/// greeting to stdout.
+fn async_hello_world() -> Future<Result<u8>>;
```

Comment:
why not return void?

Reply:
not supported yet, will do that tomorrow